src/WooCommerceServiceProvider.php

```diff
@@ -43,10 +43,11 @@ public function bindFilters()
         $woocommerce = $this->app['woocommerce'];
 
         add_filter('template_include', [$woocommerce, 'templateInclude'], 11);
-        add_filter('woocommerce_locate_template', [$woocommerce, 'template']);
-        add_filter('wc_get_template_part', [$woocommerce, 'template']);
+        add_filter('woocommerce_locate_template', [$woocommerce, 'template'], 10, 2);
+        add_filter('woocommerce_locate_core_template', [$woocommerce, 'template'], 10, 2);
```

Comment:
Wasnt aware this functionality even existed! Where is the screen to copy templates to theme, I cannot find it even if I see it in the code 😄

Reply:
It's actually used mainly for Emails template, there's copy button.